backend/layers/thirdparty/schema_validator_provider.py

```diff
@@ -23,17 +23,18 @@ def get_current_schema_version(self) -> str:
 
 
 class SchemaValidatorProvider(SchemaValidatorProviderInterface):
-    def validate_and_save_labels(self, input_file: str, output_file: str) -> Tuple[bool, list, bool]:
+    def validate_and_save_labels(self, input_file: str, output_file: str, n_workers: int) -> Tuple[bool, list, bool]:
```

Comment:
make `n_workers` Optional.
```suggestion
    def validate_and_save_labels(self, input_file: str, output_file: str, **kwargs) -> Tuple[bool, list, bool]:
```

Also update `SchemaValidatorProviderInterface(Protocol)` to match